src/move/econia/sources/user.move

```diff
@@ -1438,6 +1508,108 @@ module econia::user {
             user_address, market_id, custodian_id, quote_coins);
     }
 
+    /// Emit fill event to specified handle, if handle exists.
+    public(friend) fun emit_fill_event_for_maker_and_taker(
+        event: FillEvent
+    ) acquires MarketEventHandles {
+        emit_fill_event(event, true);
+        emit_fill_event(event, false);
+    }
+
+    public(friend) fun emit_limit_order_event(
+        event: LimitOrderEvent
+    ) acquires MarketEventHandles {
+        let user_address = event.user;
+        if (exists<MarketEventHandles>(user_address)) {
+            let market_event_handles_ref_mut =
+                borrow_global_mut<MarketEventHandles>(user_address);
+            let limit_order_events_ref_mut =
+                &mut market_event_handles_ref_mut.limit_order_events;
+            let market_account_id =
+                get_market_account_id(event.market_id, event.custodian_id);
+            let has_handle = table::contains(
+                limit_order_events_ref_mut, market_account_id);
+            if (has_handle) {
+                let limit_order_event_handle_ref_mut = table::borrow_mut(
+                    limit_order_events_ref_mut, market_account_id);
+                event::emit_event(limit_order_event_handle_ref_mut, event)
+            }
+        }
+    }
+
+    public(friend) fun get_fill_event(
```

Comment:
Name should not begin with `get`, as this is not a getter. `create`?

Reply:
58a7aa9 renames to `pack_fill_event()`